src/core/dbus.c

```diff
@@ -979,6 +989,19 @@ int bus_init_private(Manager *m) {
         return 0;
 }
 
+static int bus_track_stash(sd_bus_track *t, char ***l) {
+        for (const char *n = sd_bus_track_first(t); n; n = sd_bus_track_next(t)) {
+                int c;
+
+                c = sd_bus_track_count_name(t, n);
+                for (int j = 0; j < c; j++)
+                        if (strv_extend(l, n) < 0)
```

Comment:
in systemd we prefer to have output params that are only initialized on success. i.e. please build the strv locally first, and then on access move them into a `ret` parameter.